framework/include/euv_pipe.h

```diff
@@ -26,10 +26,18 @@ typedef enum {
     EUV_PIPE_TYPE_CLIENT_RPMSG,
 } euv_pipe_mode_t;
 
+typedef enum {
+    EUV_ALL_PIPE_CLOSED = 0x00,
```

Comment:
It is recommended to use bit offset operations instead of hexadecimal to avoid conflicts with decimal. like 0x01<N